container/features/src/main/resources/features.xml

```diff
@@ -328,6 +329,12 @@
         <bundle>mvn:org.opennms.core/org.opennms.core.xml/${project.version}</bundle>
         <bundle>mvn:org.opennms/opennms-util/${project.version}</bundle>
     </feature>
+
+    <feature name="configservice" version="${project.version}" description="OpenNMS :: Feature :: Configservice">
```

Comment:
@j-white ,
I am getting a
`Caused by: org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type [org.opennms.features.distributed.kvstore.api.BlobStore] found for dependency: expected at least 1 bean which qualifies as autowire candidate for this dependency. Dependency annotations: {}
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.raiseNoSuchBeanDefinitionException(DefaultListableBeanFactory.java:1380)
`
I tried to add the opennms-blobstore-postgres to the feature and add the configservice to the core feature but it doesn't seem to solve the problem. Probably I am missing something else? It worked when trying the json store without editing this file...